stories/topics/proc-aap-aws-modify-alb-cidr-range.adoc

```diff
@@ -3,7 +3,7 @@
 = Modify Load Balancers CIDR Range
 
 == Default Load Balancers CIDR Range Configuration on Ansible Automation Platform
-The default CIDR range on Load Balancers are set to 0.0.0.0/0, allowing everyone to access the Load Balancers. If you are looking to update these CIDR range, you can update them before or after the deployment. You can follow the steps below to get this done.
+The default CIDR range on Load Balancers are set to 0.0.0.0/0, allowing everyone to access the Load Balancers. If you are looking to update these CIDR ranges, you can choose to update them either before or after the deployment. You can follow the steps below to get this done.
 
 === Procedure - update before the deployment
 . In the AWS Console, navigate to *CloudFormation* and click on your Red Hat Ansible Automation Platform deployment.
```

Comment:
Change line 8 as follows.

===Updating before the deployment

.Procedure

Reply:
Resolved!